### PR TITLE
Update gazebo gripper plugin

### PR DESCRIPTION
**Pull request by ibaiape**
Reduced both plugins to a single plugin using PID position control.

Updated plugin to take in multiplier values for specific joints (substituting function of `mimic` plugin).

Renamed plugin.

### CODE REVIEW

File: robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp [hunk 1]
```diff
@@ -114,6 +114,27 @@ namespace gazebo
...
     createGenericTopics(node_name);
 
+    if (_sdf->HasElement("p")){
+      kp = _sdf->GetElement("p")->Get<double>();
+      RCLCPP_INFO(ros_node_->get_logger(), "Taking kp value of [%s] from plugin", std::to_string(kp).c_str());
+    }else{
+      RCLCPP_INFO(ros_node_->get_logger(), "Using default kp value of [%s]", std::to_string(kp).c_str());
+    }
+
+    if (_sdf->HasElement("i")){
```
Comment (ahcorde) [inline robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp]: I would call this `ki`

File: robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp [hunk 2]
```diff
@@ -114,6 +114,27 @@ namespace gazebo
...
+      ki = _sdf->GetElement("i")->Get<double>();
+      RCLCPP_INFO(ros_node_->get_logger(), "Taking ki value of [%s] from plugin", std::to_string(ki).c_str());
+    }else{
+      RCLCPP_INFO(ros_node_->get_logger(), "Using default ki value of [%s]", std::to_string(ki).c_str());
+    }
+
+    if (_sdf->HasElement("d")){
```
Comment (ahcorde) [inline robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp]: I would call this `kd`

File: robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp [hunk 3]
```diff
@@ -114,6 +114,27 @@ namespace gazebo
 
     createGenericTopics(node_name);
 
+    if (_sdf->HasElement("p")){
```
Comment (ahcorde) [inline robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp]: I would call this `kp`

File: robotiq_hande_gripper_description/urdf/robotiq_hande.urdf.xacro [hunk 4]
```diff
@@ -84,7 +84,10 @@
     </gazebo>
 
     <gazebo>
-      <plugin filename="librobotiq_linear_gripper_gazebo_plugin.so" name="robotiq_hande_gripper">
+      <plugin filename="librobotiq_gripper_gazebo_plugin.so" name="robotiq_hande_gripper">
+        <p>10</p>
```
Comment (ahcorde) [inline robotiq_hande_gripper_description/urdf/robotiq_hande.urdf.xacro]: I would call this `ki`, `kp` and `kd`

File: robotiq_hande_gripper_description/urdf/robotiq_hande.urdf [hunk 5]
```diff
@@ -83,7 +83,10 @@
     <selfCollide>True</selfCollide>
   </gazebo>
   <gazebo>
-    <plugin filename="librobotiq_linear_gripper_gazebo_plugin.so" name="robotiq_hande_gripper">
+    <plugin filename="librobotiq_gripper_gazebo_plugin.so" name="robotiq_hande_gripper">
+      <p>10</p>
```
Comment (ahcorde) [inline robotiq_hande_gripper_description/urdf/robotiq_hande.urdf]: I would call this `ki`, `kp` and `kd`

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp (hunk 1 above)
Comment (ibaiape) [reply robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp]: https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/2/commits/1e1314ecce4bf52483e366a87910af57a9017bd6

File: robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp (hunk 2 above)
Comment (ibaiape) [reply robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp]: https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/2/commits/1e1314ecce4bf52483e366a87910af57a9017bd6

File: robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp (hunk 3 above)
Comment (ibaiape) [reply robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp]: https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/2/commits/1e1314ecce4bf52483e366a87910af57a9017bd6

File: robotiq_hande_gripper_description/urdf/robotiq_hande.urdf.xacro (hunk 4 above)
Comment (ibaiape) [reply robotiq_hande_gripper_description/urdf/robotiq_hande.urdf.xacro]: https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/2/commits/1e1314ecce4bf52483e366a87910af57a9017bd6

File: robotiq_hande_gripper_description/urdf/robotiq_hande.urdf (hunk 5 above)
Comment (ibaiape) [reply robotiq_hande_gripper_description/urdf/robotiq_hande.urdf]: https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/2/commits/1e1314ecce4bf52483e366a87910af57a9017bd6

**Review by ahcorde — APPROVED** [review]

**Pull request merged**